csv.go

```diff
@@ -58,42 +56,22 @@ func csvMain(_ *cobra.Command, args []string) error {
 		licenseURL := "Unknown"
 		licenseName := "Unknown"
 		if lib.LicensePath != "" {
-			// Find a URL for the license file, based on the URL of a remote for the Git repository.
-			var errs []string
-			repo, err := licenses.FindGitRepo(lib.LicensePath)
-			if err != nil {
-				// Can't find Git repo (possibly a Go Module?) - derive URL from lib name instead.
-				lURL, err := lib.FileURL(lib.LicensePath)
-				if err != nil {
-					errs = append(errs, err.Error())
-				} else {
-					licenseURL = lURL.String()
-				}
+			name, _, err := classifier.Identify(lib.LicensePath)
+			if err == nil {
+				licenseName = name
 			} else {
-				for _, remote := range gitRemotes {
-					url, err := repo.FileURL(lib.LicensePath, remote)
-					if err != nil {
-						errs = append(errs, err.Error())
-						continue
-					}
-					licenseURL = url.String()
-					break
-				}
-			}
-			if licenseURL == "Unknown" {
-				glog.Errorf("Error discovering URL for %q:\n- %s", lib.LicensePath, strings.Join(errs, "\n- "))
-			}
-			licenseName, _, err = classifier.Identify(lib.LicensePath)
-			if err != nil {
 				glog.Errorf("Error identifying license in %q: %v", lib.LicensePath, err)
-				licenseName = "Unknown"
+			}
+			url, err := lib.FileURL(context.Background(), lib.LicensePath)
+			if err == nil {
+				licenseURL = url
+			} else {
+				glog.Warningf("Error discovering license URL: %s", err)
 			}
 		}
-		// Remove the "*/vendor/" prefix from the library name for conciseness.
-		if err := writer.Write([]string{unvendor(lib.Name()), licenseURL, licenseName}); err != nil {
+		if _, err := fmt.Fprintln(os.Stdout, strings.Join([]string{lib.Name(), licenseURL, licenseName}, ",")); err != nil {
```

Comment:
nit: i'd just revert to the previous `csv.Writer` behavior.

There's a few other quirks in CSV behavior that while I'm not 100% sure we need to worry about, it's easier to just trust the csv library to handle it properly instead of rolling our own.

Reply:
No problem, just reverted this.